apps/charterafrica/src/components/Explainers/Explainer.js

```diff
@@ -0,0 +1,70 @@
+import RichTypography from "@/commons-ui/core/RichTypography";
+import { Figure } from "@commons-ui/next";
+import { Grid, Divider } from "@mui/material";
+import PropTypes from "prop-types";
+import React from "react";
+
+import RichText from "../RichText";
+
+function Explainer({ image, description, title }) {
+  return (
+    <>
+      <Grid display="flex" justifyContent="center" sx={{ p: 8 }} container>
+        <Grid container spacing={4}>
```

Comment:
```suggestion
```

1. Don't think we need this.
2. After container, there must be item... Can't have Grid container inside Grid container without Grid item in between.